src/zino/statemodels.py

```diff
@@ -13,13 +13,39 @@
 IPAddress = Union[IPv4Address, IPv6Address]
 AlarmType = Literal["yellow", "red"]
 PortOrIPAddress = Union[int, IPAddress, AlarmType]
-BgpStyle = Literal["juniper", "cisco", "general"]
-BgpAdminStatus = Literal["running", "halted", "start", "stop"]
-BgpOperState = Literal["idle", "connect", "active", "opensent", "openconfirm", "established", "down"]
 
 _logger = logging.getLogger(__name__)
 
 
+class BGPStyle(StrEnum):
+    """The set of allowable BGP styles"""
+
+    JUNIPER = "juniper"
+    CISCO = "cisco"
+    GENERAL = "general"
+
+
+class BGPAdminStatus(StrEnum):
+    """The set of allowable BGP admin status"""
+
+    RUNNING = "running"
+    HALTED = "halted"
+    START = "start"
+    STOP = "stop"
+
+
+class BGPOperStates(StrEnum):
```

Comment:
To pluralize or not to pluralize, that's the question.

Reply:
Yes, good point. An instance of this represent a single state, not multiple, so the class should really have a non-plural name